packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -27,19 +27,18 @@ import { addQueryArgs } from '@wordpress/url';
 /**
  * Module constants
  */
+const MAX_TERMS_SUGGESTIONS = 20;
 const DEFAULT_QUERY = {
-	per_page: -1,
+	per_page: MAX_TERMS_SUGGESTIONS,
 	orderby: 'count',
 	order: 'desc',
 	_fields: 'id,name',
-};
-const MAX_TERMS_SUGGESTIONS = 20;
+}
```

Comment:
Add `;` to fix linting errors.
```suggestion
};
```